Sources/Foundation/UserDefaults.swift

```diff
@@ -50,7 +48,7 @@ open class UserDefaults: NSObject {
             return true
         }
         
-        let isOfCommonTypes =  value is String || value is Data || value is Date || value is Int || value is Bool || value is CGFloat
+        let isOfCommonTypes = value is String || value is Data || value is Date || value is Int || value is Bool || value is CGFloat
```

Comment:
I wanted to reconfigure such long lines of code as follows, but I want to ask first. Are you writing such long lines of code because of performance anxiety? Or is that how it was forgotten?

```suggestion
        let isOfCommonTypes = value is String 
            || value is Data 
            || value is Date 
            || value is Int 
            || value is Bool 
            || value is CGFloat
```

Reply:
🤔IMO leaving it alone is okay because because this line was really simple and easy. Splitting it into multiple lines wouldn’t affect anything (including performance) since it even makes no difference in SIL, but may make the single statement taking up unnecessarily large visual space.